v22.2/alter-backup-schedule.md

```diff
@@ -0,0 +1,190 @@
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} The `ALTER BACKUP SCHEDULE` statement modifies an existing [backup schedule](manage-a-backup-schedule.html). You can use `ALTER BACKUP SCHEDULE` to do the following:
+
+- Set a different name for a backup schedule.
+- Change a scheduled backup's storage location.
+- Apply additional backup options or schedule options to backups and schedules.
+- Adjust the cadence and type of scheduled backups.
+
+## Required privileges
+
+To alter a backup schedule, you must be the owner of the backup schedule, i.e., the user that [created the backup schedule](create-schedule-for-backup.html).
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/master/grammar_svg/alter_backup_schedule.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+-------------
+`schedule_id` | The schedule's ID that `CREATE SCHEDULE FOR BACKUP` and `SHOW SCHEDULES` display.
+`schedule_label` | The name or label given to the backup schedule.
+`collectionURI` | The URI where you want to store the backup. See [Backup file URLs](backup.html#backup-file-urls) for detail on forming the URI.
+`option` | Control the backup behavior with a comma-separated list of these [options](#backup-options).
+`RECURRING crontab` | Specify when the backup should be taken. By default, these are incremental backups. A separate schedule may be created automatically to write full backups at a regular cadence, depending on the frequency of the incremental backups. You can likewise modify this separate schedule with `ALTER BACKUP SCHEDULE`. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron). All times in UTC. <br><br>Example: `'@daily'` (run daily at midnight)
+`FULL BACKUP crontab / ALWAYS` | Specify when to take a new full backup. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron) or as `ALWAYS`. <br><br>`FULL BACKUP ALWAYS` will trigger `RECURRING` to always take full backups. **Note:** If you do not have an Enterprise license then `ALWAYS` is the only accepted value of `FULL BACKUP`. <br><br>If you omit the `FULL BACKUP` clause, the default backup schedule will be as follows: <ul><li>`RECURRING` <= 1 hour: Default to `FULL BACKUP '@daily'`</li><li>`RECURRING` <= 1 day: Default to `FULL BACKUP '@weekly'`</li><li>Otherwise: Default to `FULL BACKUP ALWAYS`</li></ul>
+`schedule_option` | Control the schedule behavior with a comma-separated list of these [schedule options](#schedule-options).
+
+### Backup options
+
+You can use the backup options in this table to control the behavior of your backups. See [Apply different options to scheduled backups](#apply-different-options-to-scheduled-backups) for an example.
+
+{% include {{ page.version.version }}/backups/backup-options-for-schedules.md %}
+
+### Schedule options
+
+{{site.data.alerts.callout_danger}}
```

Comment:
Question: This seems like a strange place for this warning; is there somewhere else this should live? For example, this table lists all the params to `schedule_option`, maybe the warning should be there? Or is this more conceptually: "altering backups is new in v22.2 and in preview currently" and thus really applies to this whole page: `alter-backup-schedule.md`, so maybe this warning belongs up top? I'm not sure, but seeing it here comes as a surprise.

Reply:
It specifically refers to the schedule options.... these have been experimental for a while and the team wants to keep them as such, i.e., preview. I will clarify that schedule options is in preview within the warning box.